src/calculateRentalCost.js

```diff
@@ -28,7 +28,28 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const value = 40;
+  let sum = 0;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+
+  if (days >= 7) {
+    sum = days * value - bigDiscount;
+
+    return sum;
+  }
+
+  if (days >= 3) {
+    sum = days * value - smallDiscount;
+
+    return sum;
+  }
+
+  if (days >= 1) {
```

Comment:
The last condition 'if (days >= 1)' is not needed. You can remove it and return the value directly in the last line of the function.